examples/document/basic-example.html

```diff
@@ -30581,11 +30580,13 @@ <h2 id="_conclusion">Conclusion</h2>
 
 	let ready = new Promise(function(resolve, reject){
 		if (document.readyState === "interactive" || document.readyState === "complete") {
+      console.log(document.readyState)
```

Comment:
Could you please regenerate this file so the generated file is consistent with the code